manifesttestutils/util.go

```diff
@@ -314,7 +314,7 @@ func mustReadManifest(relativePathToManifest string) string {
 }
 
 var (
-	yamlSeparator = regexp.MustCompile("\n---")
+	yamlSeparator = regexp.MustCompile("\n---\n")
```

Comment:
nit: What if we defined this once here, exported it, and used it in helmchart/manifests.go?

Reply:
340185c